AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.h

```diff
@@ -3,13 +3,55 @@
 
 #import <Foundation/Foundation.h>
 
-#import "MSDBStorage.h"
-#import "MSDataStore.h"
-#import "MSDocumentStore.h"
+#import "MSBaseOptions.h"
+#import "MSDocumentWrapper.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface MSDBDocumentStore : NSObject <MSDocumentStore>
+@interface MSDBDocumentStore : NSObject
+
+/**
+ * Create or replace an entry in the store.
+ *
+ * @param partition Document partition.
+ * @param documentWrapper Document wrapper object to store (may be `nil`, e.g. for a DELETE operation).
```

Comment:
Since this API is only for create and replace we don't need this "(may be `nil`, e.g. for a DELETE operation)."